okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -254,13 +252,9 @@ open class OkHttpClient internal constructor(
   /** Web socket and HTTP/2 ping interval (in milliseconds). By default pings are not sent. */
   fun pingIntervalMillis(): Int = pingInterval
 
-  internal fun internalCache(): InternalCache? {
-    return cache?.internalCache ?: internalCache
-  }
-
   /** Prepares the [request] to be executed at some point in the future. */
   override fun newCall(request: Request): Call {
-    return RealCall.newRealCall(this, request, false /* for web socket */)
+    return RealCall.newRealCall(this, request, forWebSocket = false)
```

Comment:
Nice!  We should do this for all ints and bools.

Reply:
@yschimke Is someone working on it? I'd like to do it.

Reply:
@jemaystermind it's easy to screw this up because the names are noisy if the context makes their purposes clear. Starting with parameters we're already commenting would be a good start though.

Reply:
@swankjesse Good point. I'll take your advice and see what I can do. Thanks!